routes/experimenters.js

```diff
@@ -30,3 +31,39 @@ exports.create = function (req, res) {
       });
    });
 };
+
+function updateExperimenter(req, res, exp) {
```

Comment:
Let's put this function after the exports.update declaration, to keep the exports together.